.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,41 +1,63 @@
 ## PR Summary
 
-## PR Checklist
-
-- [ ] Has Pytest style unit tests
-- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
-- [ ] New features are documented, with examples if plot related
-- [ ] Documentation is sphinx and numpydoc compliant
-- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
-- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way
-
 <!--
-Thank you so much for your PR!  To help us review your contribution, please
-consider the following points:
+Thank you so much for your PR! Please summarize the purpose of the PR here
+using at least 1-2 sentences.
+
+To help us review your pull request, make sure:
+
+- The PR title descriptively summarizes the changes.
+    - e.g. "Raise `ValueError` on non-numeric input to `set_xlim`"
+    - Please don't use non-descriptive titles such as "Addresses issue #8576".
+- The PR summary includes:
+    - How the PR implements any changes.
+    - Why these changes are necessary.
+- The PR is not out of master, but out of a separate branch.
+    - e.g. `your-user-name:non-numeric-xlim -> matplotlib:master`
+- Optional: PR cross-links related issues.
+-->
 
-- A development guide is available at https://matplotlib.org/devdocs/devel/index.html.
+## PR Checklists
 
-- Help with git and github is available at
-  https://matplotlib.org/devel/gitwash/development_workflow.html.
+<!-- Feel free to delete any checkboxes that do not apply to this PR. -->
 
-- Do not create the PR out of master, but out of a separate branch.
+New code:
 
-- The PR title should summarize the changes, for example "Raise ValueError on
-  non-numeric input to set_xlim".  Avoid non-descriptive titles such as
-  "Addresses issue #8576".
+- [ ] has Pytest style unit tests (and `pytest` passes)
```

Comment:
```suggestion
- [ ] has pytest style unit tests (and `pytest` passes)
```

Reply:
> The alternative would be to drastically shorten and just link
> https://matplotlib.org/devdocs/devel/coding_guide.html#summary-for-pr-authors.

Maybe the fact that I had never seen this page is a sign of my irresponsibility, but maybe also a bad sign for discoverability of that page...